### ISSUE TITLE
Remove userInputs from language dictionaries

### ISSUE THREAD

**Issue by MrWook** [zxcvbn-ts/zxcvbn#69]
### Discussed in https://github.com/zxcvbn-ts/zxcvbn/discussions/68

<div type='discussions-op-text'>

<sup>Originally posted by **paulverbeke** September 18, 2021</sup>
Hi @MrWook hope you're doing well.
I finally got the time to play around with this library.
I'm trying to implement it in one of my project but I cannot make the userInputs dictionnary working for me.
I'm setting these options:
`const options3 = {
    translations: zxcvbnts['language-fr'].translations,
    graphs: zxcvbnts['language-common'].adjacencyGraphs,
    dictionary: {
        userInputs: ['test456123@gmail.com'],
        ...zxcvbnts['language-common'].dictionary,
        ...zxcvbnts['language-en'].dictionary,
        ...zxcvbnts['language-fr'].dictionary
    },
}
zxcvbnts.core.ZxcvbnOptions.setOptions(options3)`

calling with this password:
`zxcvbnts.core.zxcvbn("test456123@gmail.com")`

and getting this result with a score of 4 and no suggestions -_-:
`calcTime: 13
crackTimesDisplay: Object { onlineThrottling100PerHour: "siècles", onlineNoThrottling10PerSecond: "siècles", offlineSlowHashing1e4PerSecond: "siècles", … }
crackTimesSeconds: Object { onlineThrottling100PerHour: 271252803600000000, onlineNoThrottling10PerSecond: 753480010000000, offlineSlowHashing1e4PerSecond: 753480010000, … }
feedback: Object { warning: "", suggestions: [] }
guesses: 7534800100000000
guessesLog10: 15.877071734869821
password: "test456123@gmail.com"
score: 4`

What am I doing wrong here ?</div>

Basically the language dictionaries have the userInputs as a default empty array. If you add your userInputs dictionary before you import one of the other dictionaries your userInputs dictionary will be overwritten.
This is pretty confusing which means the userInput needs to be removed from the language dictionaries.

**Comment by paulverbeke** [comment]
Great thanks for this, did you remembered the reason why it was done like this in the first place ?

**Comment by MrWook** [comment]
I think it was duo some typescript types 🤔
